From 751b0a6cf3d2599d2fa4380f4e91d63df323c07c Mon Sep 17 00:00:00 2001
From: "Loc Phuoc Pham.Lam" <loc.lam.px@renesas.com>
Date: Wed, 29 May 2019 11:33:20 +0700
Subject: [PATCH 1/2] plat: renesas: rcar: qos: E3: mstat390.h: Modify default
 setting for RZ/G2E

This patch file disables 6 unused modules:
 + IMR
 + RealTime Core
 + RT-DMAC
 + Secure Engine
 + ADSP
 + DCU
for default setting to be suitable for RZ/G2-E

Signed-off-by: Loc Phuoc Pham.Lam <loc.lam.px@renesas.com>
---
 .../renesas/rcar/qos/E3/qos_init_e3_v10_mstat390.h | 32 +++++++++++-----------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/plat/renesas/rcar/qos/E3/qos_init_e3_v10_mstat390.h b/plat/renesas/rcar/qos/E3/qos_init_e3_v10_mstat390.h
index e4a3f7a..b02fca5 100644
--- a/plat/renesas/rcar/qos/E3/qos_init_e3_v10_mstat390.h
+++ b/plat/renesas/rcar/qos/E3/qos_init_e3_v10_mstat390.h
@@ -21,20 +21,20 @@ static uint64_t mstat_fix[]={
 	/* 0x0098, */ 0x0000000000000000UL,
 	/* 0x00a0, */ 0x000C08380000FFFFUL,
 	/* 0x00a8, */ 0x000C04110000FFFFUL,
-	/* 0x00b0, */ 0x000C04110000FFFFUL,
+	/* 0x00b0, */ 0x000C04150000FFFFUL,
 	/* 0x00b8, */ 0x0000000000000000UL,
 	/* 0x00c0, */ 0x000C08380000FFFFUL,
 	/* 0x00c8, */ 0x000C04110000FFFFUL,
-	/* 0x00d0, */ 0x000C04110000FFFFUL,
+	/* 0x00d0, */ 0x000C04150000FFFFUL,
 	/* 0x00d8, */ 0x0000000000000000UL,
 	/* 0x00e0, */ 0x0000000000000000UL,
 	/* 0x00e8, */ 0x0000000000000000UL,
-	/* 0x00f0, */ 0x001018580000FFFFUL,
-	/* 0x00f8, */ 0x000C04400000FFFFUL,
+	/* 0x00f0, */ 0x0000000000000000UL,
+	/* 0x00f8, */ 0x000C084F0000FFFFUL,
 	/* 0x0100, */ 0x0000000000000000UL,
 	/* 0x0108, */ 0x0000000000000000UL,
-	/* 0x0110, */ 0x001008580000FFFFUL,
-	/* 0x0118, */ 0x000C19660000FFFFUL,
+	/* 0x0110, */ 0x0000000000000000UL,
+	/* 0x0118, */ 0x000C21E40000FFFFUL,
 	/* 0x0120, */ 0x0000000000000000UL,
 	/* 0x0128, */ 0x0000000000000000UL,
 	/* 0x0130, */ 0x0000000000000000UL,
@@ -60,12 +60,12 @@ static uint64_t mstat_fix[]={
 	/* 0x01d0, */ 0x0000000000000000UL,
 	/* 0x01d8, */ 0x0000000000000000UL,
 	/* 0x01e0, */ 0x0000000000000000UL,
-	/* 0x01e8, */ 0x000C04020000FFFFUL,
-	/* 0x01f0, */ 0x000C04090000FFFFUL,
+	/* 0x01e8, */ 0x0000000000000000UL,
+	/* 0x01f0, */ 0x0000000000000000UL,
 	/* 0x01f8, */ 0x0000000000000000UL,
 	/* 0x0200, */ 0x0000000000000000UL,
-	/* 0x0208, */ 0x000C04090000FFFFUL,
-	/* 0x0210, */ 0x000C04090000FFFFUL,
+	/* 0x0208, */ 0x0000000000000000UL,
+	/* 0x0210, */ 0x0000000000000000UL,
 	/* 0x0218, */ 0x0000000000000000UL,
 	/* 0x0220, */ 0x0000000000000000UL,
 	/* 0x0228, */ 0x0000000000000000UL,
@@ -75,10 +75,10 @@ static uint64_t mstat_fix[]={
 	/* 0x0248, */ 0x0000000000000000UL,
 	/* 0x0250, */ 0x0000000000000000UL,
 	/* 0x0258, */ 0x0000000000000000UL,
-	/* 0x0260, */ 0x000C0C2A0000FFFFUL,
+	/* 0x0260, */ 0x0000000000000000UL,
 	/* 0x0268, */ 0x001410040000FFFFUL,
 	/* 0x0270, */ 0x001404020000FFFFUL,
-	/* 0x0278, */ 0x000C08110000FFFFUL,
+	/* 0x0278, */ 0x0000000000000000UL,
 	/* 0x0280, */ 0x0000000000000000UL,
 	/* 0x0288, */ 0x0000000000000000UL,
 	/* 0x0290, */ 0x001410040000FFFFUL,
@@ -86,13 +86,13 @@ static uint64_t mstat_fix[]={
 	/* 0x02a0, */ 0x000C04090000FFFFUL,
 	/* 0x02a8, */ 0x000C04090000FFFFUL,
 	/* 0x02b0, */ 0x0000000000000000UL,
-	/* 0x02b8, */ 0x000C04020000FFFFUL,
+	/* 0x02b8, */ 0x0000000000000000UL,
 	/* 0x02c0, */ 0x0000000000000000UL,
 	/* 0x02c8, */ 0x0000000000000000UL,
 	/* 0x02d0, */ 0x000C04090000FFFFUL,
 	/* 0x02d8, */ 0x000C04090000FFFFUL,
 	/* 0x02e0, */ 0x0000000000000000UL,
-	/* 0x02e8, */ 0x000C04020000FFFFUL,
+	/* 0x02e8, */ 0x0000000000000000UL,
 	/* 0x02f0, */ 0x0000000000000000UL,
 	/* 0x02f8, */ 0x0000000000000000UL,
 	/* 0x0300, */ 0x0000000000000000UL,
@@ -178,11 +178,11 @@ static uint64_t mstat_be[]={
 	/* 0x01d8, */ 0x0000000000000000UL,
 	/* 0x01e0, */ 0x0000000000000000UL,
 	/* 0x01e8, */ 0x0000000000000000UL,
-	/* 0x01f0, */ 0x0021010005F79801UL,
+	/* 0x01f0, */ 0x0000000000000000UL,
 	/* 0x01f8, */ 0x0000000000000000UL,
 	/* 0x0200, */ 0x0000000000000000UL,
 	/* 0x0208, */ 0x0000000000000000UL,
-	/* 0x0210, */ 0x0021010005F79801UL,
+	/* 0x0210, */ 0x0000000000000000UL,
 	/* 0x0218, */ 0x0011010005F79801UL,
 	/* 0x0220, */ 0x0011010005F79801UL,
 	/* 0x0228, */ 0x0000000000000000UL,
-- 
1.9.1

